code/modules/projectiles/projectile/bullets.dm

```diff
@@ -294,6 +294,53 @@
 	embed = 0
 	sharp = 0
 
+/obj/item/projectile/bullet/pistol/papadov
+	name = "380"
+	damage = 20
+	agony = 0
+	embed = 0
+	sharp = 0
+	weaken = 0
+
+
+/obj/item/projectile/bullet/pistol/papadov/tranq
+	name = "380 T"
+	damage = 1
+	stun = 0
+	weaken = 0
+	drowsy = 4

```

Comment:
If I am not wrong, this will cause drowsy even if the armor does not block, that is also caused by the on hit proc.